src/GraphQL.Tests/AuthorizationTests.cs

```diff
@@ -0,0 +1,125 @@
+using GraphQL.Types;
+
+namespace GraphQL.Tests;
```

Comment:
Nice tests. I wonder how you have enough patience to write all that.

Reply:
Copy and paste!!